extensions/provisioning/creates.go

```diff
@@ -120,6 +125,35 @@ func CreateProvisioningCluster(client *rancher.Client, provider Provider, cluste
 
 	machinePools := machinepools.
 		CreateAllMachinePools(machineConfigs, pools, machinePoolResponses, provider.Roles, hostnameTruncation)
+
+	if clustersConfig.CloudProvider == provisioninginput.VsphereCloudProviderName.String() {
+
+		vcenterCredentials := map[string]interface{}{
+			"datacenters": machinePoolConfigs[0].Object["datacenter"],
+			"host":        cloudCredential.VmwareVsphereConfig.Vcenter,
+			"password":    vsphere.GetVspherePassword(),
+			"username":    cloudCredential.VmwareVsphereConfig.Username,
+		}
+		clustersConfig.AddOnConfig = &provisioninginput.AddOnConfig{
+			ChartValues: &rkev1.GenericMap{
+				Data: map[string]interface{}{
+					"rancher-vsphere-cpi": map[string]interface{}{
+						"vCenter": vcenterCredentials,
+					},
+					"rancher-vsphere-csi": map[string]interface{}{
+						// "csiWindowsSupport": map[string]interface{}{
```

Comment:
Was this intended to be uncommented or cleaned up?

Reply:
let me remove that for now - there is a separate issue with windows right now blocking these charts from properly installing. Once that's in, I'll have to open a new PR with windows support.